prime/src/main/java/com/telenordigital/prime/ocs/OcsState.java

```diff
@@ -11,12 +11,14 @@
 import java.util.Map;
 
 import static com.google.common.base.Preconditions.checkNotNull;
+import static com.telenordigital.prime.disruptor.PrimeEventMessageType.*;
 
 public final class OcsState implements EventHandler<PrimeEvent> {
 
     private static final Logger LOG = LoggerFactory.getLogger(OcsState.class);
 
     private final Map<String, Long> dataPackMap = new HashMap<>();
+    private final Map<String, Long> bucketReservedMap = new HashMap<>();
```

Comment:
This also needs evolution in future. The ocs-api will now support different types of bundle consumptions, but the in-memory bundles are still fixed to just one type - DATA.